app/views/idv/doc_auth/agreement.html.erb

```diff
@@ -19,14 +19,17 @@
                        url: url_for,
                        method: 'put',
                        html: { autocomplete: 'off', class: 'margin-top-2 js-consent-continue-form' } do |f| %>
-  <br/>
-  <label class="margin-top-neg-1 margin-bottom-4" for="ial2_consent_given">
-    <div class="checkbox">
-      <%= check_box_tag :ial2_consent_given, true, false %>
-      <span class="indicator"></span>
-      <%= t('doc_auth.instructions.consent') %>
-      <%= new_window_link_to t('doc_auth.instructions.learn_more'), 'https://login.gov/policy/' %>
-    </div>
+  <%= check_box_tag(
+        :ial2_consent_given,
+        true,
+        false,
+        class: 'usa-checkbox__input',
+      ) %>
+  <label
+    class="usa-checkbox__label margin-bottom-4"
+    for="ial2_consent_given"
+  ><%= t('doc_auth.instructions.consent') %>
+    <%= new_window_link_to t('doc_auth.instructions.learn_more'), 'https://login.gov/policy/' %>
```

Comment:
I know this URL was already like this but can we use our method for this?

```suggestion
    <%= new_window_link_to t('doc_auth.instructions.learn_more'), MarketingSite.security_and_privacy_practices_url %>
```

Reply:
> I know this URL was already like this but can we use our method for this?

Good call, updated in d58f2e18f.